src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala

```diff
@@ -105,7 +105,7 @@ object Column {
       bundles.map { bundle =>
         bundle.bundleInstallations.map { node =>
           val nodeIpAndPort = node.uniqueAddress.address.toString.dropWhile(_ != '@').drop(1)
-          if (bundle.bundleExecutions.map(_.host) contains node.uniqueAddress.address.host.get)
+          if (bundle.bundleExecutions.filter(_.isStarted).map(_.host) contains node.uniqueAddress.address.host.get)
```

Comment:
Yes. But only the values left out after the filter. :) Okay, I can change to collect.